README.md

```diff
@@ -421,27 +424,39 @@ For example, in XML:
       - name: server2
         number: 3390
     </option:ports>
-    <option:tests>
+    <option:tests><![CDATA[
       - |
         #!/bin/bash
         # Check for login on vm1
         logins=$(ssh vm1 last root | grep root | wc -l)
-        [ $logins -ge 1 ] || exit 1
+        if [ $logins -lt 1 ]; then
+          # Output a hint to stderr
+          echo "You haven't logged in to vm1, yet." >&2
+          exit 1
+        fi
         exit 0
       - |
         #!/bin/bash
-        # Check for login on vm2
-        logins=$(ssh vm2 last root | grep root | wc -l)
-        [ $logins -ge 1 ] || exit 1
+        # Check for file
+        file=foobar
+        if [ ! -e ${file} ]; then
+          # Output a hint to stderr
+          echo "File \"${file}\" doesn't exist." >&2
+          exit 1
+        fi
         exit 0
-    </option:tests>
+    ]]></option:tests>
   </hastexo>
 </vertical>
 ```
 
 **Important**: Do this only *once per section*. Defining it more that once
 per section is not supported.
 
+Note on tests: as seen in the above example, it is recommended to wrap them all
+in `<![CDATA[..]]>` tags.  This avoids XML parsing errors when special
+characters are encoutered, such as the `&` used to output to stderr in bash.
```

Comment:
Typo, should be "encountered".

Also, factually misleading: `&` sends a process to the background. I presume you meant the `&` in `>&2` but you'll have to reference `>&2` in full here.